tests/integration/downstream/downstream_test_case.py

```diff
@@ -0,0 +1,38 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+from app.submitter.submitter import SubmitterFactory, Submitter
+from app.submitter.converter import Converter
+
+
+class MyMockSubmitter(Submitter):
+    def __init__(self):
+        self._message = None
+        self._submitted_at = None
+
+    def send_answers(self, user, metadata, schema, answers):
+        message, submitted_at = Converter.prepare_answers(user, metadata, schema, answers)
+        self._message = message
+        self._submitted_at = submitted_at
+        return submitted_at
+
+
+class DownstreamTestCase(IntegrationTestCase):
+    '''
+    Overrides the application SubmitterFactory to provide our own which
+    allows the message to be captured and interrogated.
+    '''
+    _submitter = MyMockSubmitter()
+
+    @staticmethod
+    def create_submitter():
```

Comment:
This method implies it creates a new submitter when actually it just returns the single instance, this would be better name get_submitter() or better still just make the submitter variable public?